bin/compile

```diff
@@ -48,20 +42,12 @@ if [ ! `which msgfmt` ]; then
 fi
 
 build-step "cleanup"
-# Make sure the cache is empty
-rm -rf $GETTEXT_CACHE/*
-
-# Store a copy of it in the cache so it doesn't have to be fetched again
-cp -R $GETTEXT_HOME/* $GETTEXT_CACHE/
 
 # Check for an essential binary to make sure it's installed
 if [ ! `which msgfmt` ]; then
     build-warn "Gettext installation failed"
     exit 1
 fi
 
-mkdir -p $BUILD_DIR/.profile.d
-cat <<EOF >$BUILD_DIR/.profile.d/gettext.sh
-export PATH=$HOME/.gettext/bin:$PATH
-export LD_LIBRARY_PATH=$HOME/.gettext/lib:$LD_LIBRARY_PATH
-EOF
+build-step "run msgfmt"
+find $BUILD_DIR -name \*.po -execdir msgfmt django.po -o django.mo \;
```

Comment:
@codingjoe IMHO this buildpack should only take care of getting gettext into the slug/cache, nothing else.